### PR TITLE
Add eBPF workgroup info to README.md

### PR DESCRIPTION
**Pull request by yonch**

### CODE REVIEW

**Comment by yonch** [comment]
fyi I sent the CLA for approval in the org.

File: README.md [hunk 1]
```diff
@@ -122,6 +122,7 @@ Specification: Metrics|Every Tuesday at 09:00 PT; Every Thursday alternating bet
 Specification: Sampling|Every Thursday at 08:00 PT|[Google Doc](https://docs.google.com/document/d/1gASMhmxNt9qCa8czEMheGlUW2xpORiYoD7dBD7aNtbQ/)|[Zoom](https://zoom.us/j/6729396170)||
 Swift: SDK|Every Thursday at 09:00 PT|[Google Doc](https://docs.google.com/document/d/1ShEFMywIV4LJcDYCNy41zkq8RR1sMq-tSvMBHngHcuk/edit?usp=sharing)|[Zoom](https://zoom.us/j/6729396170?pwd=UFRIL0U0VzczUk1RK0hYaXJITXVldz09)|[Slack](https://cloud-native.slack.com/archives/C01NCHR19SB)||
 Website|Every other week on Thursday at 13:30 ET|[Google Doc](https://docs.google.com/document/d/1wW0jLldwXN8Nptq2xmgETGbGn9eWP8fitvD5njM-xZY)|[Zoom](https://zoom.us/j/8203130519)||
+eBPF|Every week on Thursday at 10:30 PT|[Google Doc](https://docs.google.com/document/d/1pAxBekk6BadHpK717TbhYtBMOlyrgWSvp3QeYDPwwvk/edit?usp=sharing)|[Zoom](https://zoom.us/j/8287234601)|[Slack](https://cloud-native.slack.com/archives/C02AB15583A)|
 
 ## Related groups
```
Comment (SergeyKanzhelev) [inline README.md:127]: btw, it may be a good idea to add ebpf foundation https://ebpf.io/ in the list of related projects

**Review by SergeyKanzhelev — APPROVED** [review]

**Review by mtwo — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (yonch) [reply README.md:127]: 👍 added it to the "Related groups" section below

**Comment by yonch** [comment]
CLA should be ok now (for whoever might be able to hit merge)

**Comment by yonch** [comment]
Thanks @yurishkuro !

**Pull request merged**